lib/port_scanner.py

```diff
@@ -0,0 +1,45 @@
+import queue
+import socket
+from threading import Thread, Lock
+from queue import Queue
```

Comment:
`queue` is imported twice. Get rid of the first import

Reply:
okay